projects/storefrontlib/src/layout/launch-dialog/services/launch-dialog.service.ts

```diff
@@ -26,13 +29,21 @@ export class LaunchDialogService {
    * @param caller LAUNCH_CALLER
    * @param vcr View Container Ref of the container for inline rendering
    */
-  launch(caller: LAUNCH_CALLER, vcr?: ViewContainerRef): void {
+  launch(
+    caller: LAUNCH_CALLER | string,
+    vcr?: ViewContainerRef
+  ): void | Observable<ComponentRef<any>> {
     const config = this.findConfiguration(caller);
-    const renderer = this.getStrategy(config);
+    if (config) {
+      const renderer = this.getStrategy(config);
 
-    // Render if the strategy exists
-    if (renderer) {
-      renderer.render(config, caller, vcr);
+      // Render if the strategy exists
+      if (renderer) {
+        this._dialogClose.next(undefined);
+        return renderer.render(config, caller, vcr);
+      }
+    } else if (isDevMode) {
```

Comment:
Oops, forgot to call the function :)

Reply:
Funny thing this does actually work without the function call for some reason